src/libImaging/Unpack.c

```diff
@@ -1065,6 +1065,30 @@ unpackBGRA(UINT8 *_out, const UINT8 *in, int pixels) {
     }
 }
 
+static void
+unpackBGRA16L(UINT8 *_out, const UINT8 *in, int pixels) {
+    int i;
+    /* 16-bit RGBA, little-endian order, reversed words */
```

Comment:
```suggestion
    /* 16-bit RGBA, little-endian order, reversed bytes */
```
A suggestion, so that it matches https://github.com/python-pillow/Pillow/blob/5d070222d21138d2ead002fd33fdf5adcb708941/src/libImaging/Unpack.c#L1059

Reply:
However, aren't the words (16-bit values R, G, and B) reversed in this case, but not the bytes (8-bit values, which would be half of one of the channels I guess)?

Reply:
A is staying the same place, so I don't really think it's completely reversed at all. How about "rearranged channels"? See https://github.com/gmarkall/Pillow/pull/1

Reply:
Merged, many thanks!